applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_pop/README.md

```diff
@@ -0,0 +1,6 @@
+# Test K<sub>0</sub> procedure normal consolidation with pop
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by addition of POP and subsequent multiplication with K<sub>0,NC</sub> and checked at an integration point near the bottom of the column.
```

Comment:
```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by addition of POP and subsequent multiplication with K<sub>0</sub><sup>nc</sup> and checked at an integration point near the bottom of the column.
```

Reply:
Done.

Reply:
Walked through the other tests and tried to apply the same change.